src/EnergyPlus/CMakeLists.txt

```diff
@@ -777,11 +783,11 @@ set_target_properties(energyplusapi PROPERTIES VERSION ${ENERGYPLUS_VERSION})
 
 # bring in the Python API layer right next to the executable
 add_custom_command(
-        TARGET energyplus
+        TARGET energyplusapi
```

Comment:
Setting up the API wrapper is a closer match for the shared library not the executable.  Doesn't really change anything but makes it more appropriate.